chapter_01.md

```diff
@@ -1,2 +1,550 @@
+
+On the practical side, this is all about System Integrations. How our services or the services that expose some kind of APIs talk to each other. How do they transform and move data around and how do they know which services are available to consume. 
+
+As you might guess, APIs are extremely important, but understanding who is going to consume our APIs, what is expected from these APIs and who actually depends on us is critical. 
+
+Well-defined Context Maps help a lot to plan and understand how these “isolated’ Bounded Context and teams working on them will interact on a day to day basis. 
+
+For our example the following context map would make sense:
+![chapter_01_07](images/chapter_01_07.png)
+
+This diagram depicts the relationships between the simple Bounded Context that we have for our Conference Site application. Here we can see that there is a Customer/Supplier relationship between Call for Proposals and the Conference Agenda Bounded Context. Where Call for Proposals is a consumer of the upstream service Conference Agenda. Between these two teams, there is a Partnership relationship as well, as they need to collaborate to get things done. This means that the communication between these two teams is important and they should be able to influence each other’s roadmap. 
+
+On the other hand, the relationship with the Notification service is different. Call For Proposals has an upstream relationship with the Notification Bounded context, but it will comfort with their contracts. This means that from the Call for Proposals team perspective they cannot influence or change the Notification Bounded Context APIs. This happens a lot when we have legacy systems or when this bounded context is external to our company. 
+
+
+Jumping on the practical side, while System integrations is a very broad topic, this section focuses on a very practical recommendation: “You must learn about Consumer-Driven Contact Testing”. Once again Martin Fowler has an article about this: https://martinfowler.com/articles/consumerDrivenContracts.html 
+While the topic itself is not new (previous blog post from 2006) there are very up to date tools to actually implement this in your projects, such as Spring Cloud Contracts (https://spring.io/projects/spring-cloud-contract). 
+
+With Spring Cloud Contracts, the story goes like this, first you define a contact for your APIs, this basically means what kind of request the consumer will submit and what kind of response we will provide as a service. 
```

Comment:
Once again, we should divide the implementation from the concept explanation here. I already submitted this on the PR. 
https://github.com/salaboy/manual-arquiteto-moderno/pull/1

Reply:
Again.. there is no one to one mapping with DDD, this comes as a result of using DDD.
@kmacedovarela should I merge this PR first? As your PR to my master includes all the changes that were not synced with my fork, so it is quite difficult to review all the changes in a lot of unrelated files :)

Reply:
@kmacedovarela maybe you can send your PR to the branch in my fork called `patch-2`